src/core/AudioEngine.cpp

```diff
@@ -205,18 +166,8 @@ void AudioEngine::initDevices()
 
 
 
-void AudioEngine::startProcessing(bool needsFifo)
+void AudioEngine::startProcessing()
```

Comment:
Would it make sense to inline it to the header?  Now that 90% of the functionality is gone.

Reply:
Yeah, probably.